src/utils/logger.ts

```diff
@@ -1,7 +1,7 @@
 // adapted from https://github.com/netlify/edge-bundler/blob/main/node/logger.ts
 type LogFunction = (...args: unknown[]) => void
 
-const noopLogger: LogFunction = () => {
+export const noopLogger: LogFunction = () => {
```

Comment:
Instead of using `getLogger()` from below, I find it way cleaner to directly use the noop logger in the function declaration

```ts
function myfunction (systemLog?: Logfunction = noopLogger) {}
```